Package.swift

```diff
@@ -245,7 +245,8 @@ let package = Package(
             dependencies: [
                 "Basics",
                 "PackageLoading",
-                "PackageModel"
+                "PackageModel",
+                "SourceControl",
```

Comment:
hmm that would ideally not be the case. what are we using from SourceControl?

Reply:
`PinsStore` is using `SourceControl.Revision`

Reply:
should we consider an alternative location / abstraction for this use of revision? would be very nice not to need this dependency from a modeling pov